web/pdfjs.js

```diff
@@ -13,15 +13,6 @@
  * limitations under the License.
  */
 
-// Ensure that the viewer waits for the library to complete loading,
```

Comment:
Could you extend the commit message a bit, for future reference, to indicate why we can now safely drop this? I assume this was necessary for top-level await to work, but now that we removed that we don't need it anymore?

Reply:
Sure; please check if the update is sufficient!

Reply:
Yes, this is better indeed.